crates/storage/db/src/tables/mod.rs

```diff
@@ -387,6 +387,8 @@ tables! {
     /// From HashedAddress => NibblesSubKey => Intermediate value
     table StoragesTrie<Key = B256, Value = StorageTrieEntry, SubKey = StoredNibblesSubKey>;
 
+    /// Stores current state of verkle tree.
+    table VerkleTrie<Key = Vec<u8>, Value = Vec<u8>>;
```

Comment:
I am using `Vec<u8>` as the specified type for key-value, since this is what was suggested in rust-verkle's trait impls, but need some advice here, ig will have to convert it to alloy or our own custom type